### ISSUE TITLE
False positive warnings with random files

### ISSUE THREAD

**Issue by hannob** [crocs-muni/roca#50]
When running roca-detect I see that arbitrary files sometimes give warnings of this form:
```
2018-02-01 11:13:58 [7088] WARNING Fingerprint found in modulus roca-false idx 1 
{"type": "mod-hex", "fname": "roca-false", "idx": 1, "aux": null, "n": "0x1", "marked": true, "time_years": -1, "price_aws_c4": -1}
```

In this case the file is just random garbage (example attached). But I often see that roca-detect seems to think that legit files that aren't cryptographic keys at all (pdfs, executables etc.) show up with this warning. Can these be avoided?
[roca-false.zip](https://github.com/crocs-muni/roca/files/1685058/roca-false.zip)

**Comment by ph4r05** [comment]
Yep! thanks for info.
We will take a look.

**Comment by ph4r05** [comment]
Fixed in v1.2.10. 

The problem you encountered was caused by the `mod_line` detector which processes file line by line and tries to decode it as dec / hex coded moduli. In this particular case it hit a line with 1 which was marked as positive.

The minimal modulus bitsize is set to 256 bits. Smaller numbers are not taken into account during classification.
